.github/workflows/schedule-posts.txt

```diff
@@ -0,0 +1,40 @@
+name: Publish posts scheduled for today
+
+on:
+  schedule:
+    # Run every day at 23:59 UTC or manually run
+    - cron: "30 23 * * *"
```

Comment:
Is this scheduled for 23:30 intentionally? If so (it makes sense since maybe the process takes a bit longer to start running?), you should also change the time in the comments.

Reply:
ok